### PR TITLE
build(cargo): downgrade rkyv

### PR DESCRIPTION
**Pull request by kwonoj**
**Description:**

Speculative fix.

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

**Review by kdy1 — APPROVED** [review]
Thanks!
---

swc-bump:
 - swc_common --breaking

**Review by kdy1 — CHANGES_REQUESTED** [review]
Oh ci failed

File: crates/swc_common/src/plugin/serialized.rs [hunk 1]
```diff
@@ -173,13 +173,11 @@ where
         .map_err(|_err| Error::msg("Failed to deserialize given ptr"))
 }
 
+/*
```
Comment (kwonoj) [inline crates/swc_common/src/plugin/serialized.rs:176]: @kdy1 somehow compilation fails as soon as downgrade rkyv related with deserialization logic with versionedserializable struct. This maybe partially related with the issues we're digging - tried to disable it for now until we can reliably figure out plugin workflows.

Comment (kdy1) [reply crates/swc_common/src/plugin/serialized.rs:176]: Interesting... But yeah we should disable it for now.

**Review by kdy1 — APPROVED** [review]

**Pull request merged**